elasticdl/python/tests/embedding_test_module.py

```diff
@@ -4,23 +4,83 @@
 from elasticdl.python.elasticdl.layers.embedding import Embedding
 
 
-class CustomModel(tf.keras.Model):
-    def __init__(self, output_dim=16):
-        super(CustomModel, self).__init__(name="embedding_test_model")
+class EdlEmbeddingModel(tf.keras.Model):
+    def __init__(self, output_dim=16, weights=None):
+        """
+        Arguments:
+            output_dim: An Integer. It is the output dimension of embedding
+                layers in `EdlEmbeddingModel`.
+            weights: A numpy ndarray list. Unless `weights` is None, dense
+                layer initializes its weights using `weights`.
+        """
+        super(EdlEmbeddingModel, self).__init__(
+            name="test_model_with_edl_embedding"
+        )
         self.output_dim = output_dim
+        if weights:
+            if len(weights) != 2:
+                raise ValueError(
+                    "test_model_with_edl_embedding constructor receives "
```

Comment:
test_model_with_edl_embedding -> why not "EdlEmbeddingModel"?

Reply:
Use "EdlEmbeddingModel" here.